values-schema.yaml

```diff
@@ -218,6 +218,40 @@ definitions:
         type: string
     title: Container image
     type: object
+  jobs:
+    description: |
+      You may pass an array of Job objects. The Kubernetes API accepts a Job specification, 
+      which is meant for run-to-completion apps (e.g. batch Jobs).
+    items:
+      description: |
+        You may pass a Job object. You must pass the following properties:
+
+          enabled: (type boolean)
```

Comment:
This description is not needed. The `properties` define all types.

Reply:
It seems superfluous, but when I hover over the job property I really like having an overview of the capabilities.

Reply:
<img width="662" alt="Screenshot 2020-12-10 at 13 25 11" src="https://user-images.githubusercontent.com/20700197/101772400-9cf59d80-3aeb-11eb-975b-580fd8cf0df5.png">

Reply:
<img width="670" alt="Screenshot 2020-12-10 at 13 27 17" src="https://user-images.githubusercontent.com/20700197/101772435-a67f0580-3aeb-11eb-82f7-9c661c7518f2.png">

Reply:
It isn't refined, just to give a first impression.